org.getmonero.Monero.yaml

```diff
@@ -26,6 +26,9 @@ modules:
       path: org.getmonero.Monero.desktop
     - type: file
       path: org.getmonero.Monero.svg
+    - type: file
+      path: org.getmonero.Monero.metainfo.xml
```

Comment:
has this been submitted upstream?

(this is not flatpak specific just to be clear)

Reply:
nevermind I noticed https://github.com/monero-project/monero-gui/issues/3478